.github/workflows/python-tests.yml

```
@@ -0,0 +1,26 @@
+on:
+  push:
+    branches:
+    - master
+  pull_request:
+    branches:
+    - master
+
+name: Run Tox tests on container-ci-suite
+
+jobs:
+  tox_test:
+    name: Tox test
+    strategy:
+      matrix:
+        tox_env: [py37, py38, py39, py310, lint]
```

```suggestion
        tox_env: [py37, py38, py39, py310]
```
You have a lint env here but no definition of lint environment in the tox config. You have to either remove it here or add it there.